tests/unit_test/chat/chat_test.py

```
@@ -2088,7 +2164,7 @@ def test_delete_media_from_bsp_channel_does_not_exist():
 
     assert str(exc_info.value) == "Media deletion failed: No channel found for this bot. Please configure the channel first."
 
-from datetime import datetime
+from datetime import datetime, timedelta
```

Ruff (0.15.2)

[error] Redefinition of unused `datetime` from line 2: `datetime` redefined here

Remove definition: `datetime`

(F811)